cloud/api/protobuf/kurtosis_backend_server_api.proto

```diff
@@ -139,3 +143,31 @@ message UpdateAddressArgs {
   string postal_code = 7;
   string country = 8;
 }
+
+message LockUnlockPortRequest {
+  uint32 port_number=1;
+  string instance_short_uuid=2;
+  string service_short_uuid=3;
+  string enclave_short_uuid=4;
```

Comment:
Should we make those four attributes a message named `Port` and use that in some of the requests and responses?  It could also replace `PortAndService`.